website/cue/reference/components/sources/http_scrape.cue

```diff
@@ -2,7 +2,6 @@ package metadata
 
 components: sources: http_scrape: {
 	title: "HTTP Scrape"
-	alias: "http_scrape"
```

Comment:
@neuronull `alias` here is used for renamed components 👍

Reply:
Good to know thanks :+1:

Reply:
I didn't realize this on the initial PR either, I assume you were going for the `HTTP Scrape` => `http_scrape` mapping?

Reply:
I think I was modelling it after `prometheus_scrape` cue files, which has `alias` set.